apps/src/weblab/brambleHost.js

```diff
@@ -14,11 +14,29 @@ window.requirejs.config({baseUrl: BRAMBLE_BASE_URL});
 let webLab_;
 if (parent.getWebLab) {
   webLab_ = parent.getWebLab();
-} else {
+} else if (!brambleConfig.blankLoad) {
   console.error('ERROR: getWebLab() method not found on parent');
 }
 
 function load(Bramble) {
+  if (brambleConfig.blankLoad) {
+    // Special case for checking that Bramble will load
+    // without actually passing data
+    Bramble.load(BRAMBLE_CONTAINER, {
+      url: `${BRAMBLE_BASE_URL}/index.html`
+    });
+    Bramble.on('readyStateChange', (_, newState) => {
+      if (Bramble.MOUNTABLE === newState) {
+        window.parent.postMessage(
```

Comment:
This `postMessage` isn't actually coming from inside the iframe (my original thought), but using `postMessage` still seemed like the easiest way to communicate with the parent component, assuming specifying the `targetUrl` here and checking the `origin` on the other page is secure enough.